adobe/react-spectrum
# ColorSlider, ColorWheel localization strings for color channel names

Closes localization issue and TODO identified in release testing

## ✅ Pull Request Checklist:


## 📝 Test Instructions:

1. open http://localhost:9003/?path=/story/colorslider--rgba, http://localhost:9003/?path=/story/colorslider--hsla, and http://localhost:9003/?path=/story/colorslider--hsba
2. verify color channel names change when i18n changes
3. open http://localhost:9003/?path=/story/colorwheel--default
4. inspect the input element for the color wheel
5. verify that the aria-label for the input element updates to reflect the current locale

## 🧢 Your Project:

Adobe/Accessibility
